extmod/asyncio/core.py

```diff
@@ -3,6 +3,7 @@
 
 from time import ticks_ms as ticks, ticks_diff, ticks_add
 import sys, select
+from select import POLLIN, POLLOUT
```

Comment:
alas I don't think it works to write `POLLIN = const(select.POLLIN)`. I wonder if `POLLIN = const(1); assert POLLIN == select.POLLIN` benefits performance enough that it would be worth doing. (is micropython asyncio supposed to be cpython compatible? I guess there's no guarantee of the value of POLLIN/POLLOUT constants there. But there's no `time.ticks_ms` so probably this is a non-goal)